_posts/2019-02-01-dynamic-spawn-scheduler.md

```diff
@@ -71,14 +77,16 @@ iOS | iMac Pro 2018, 18-core | 34s | 414s ↑ | 32s ≈
 iOS | Mac Pro 2013, 6-core | 39s | 414s ↑ | 36s ≈
 iOS | MacBook Pro 2015, 4-core | 39s | 450s ↑ | 36s ≈
 
-As you can see, the remote-only configuration gave us slightly better results than the local-only configuration for clean builds, but the remote-only configuration became unacceptably slow for incremental builds. As expected, though, dynamic execution wins heads-down in all cases: clean builds are many times faster than they are with local or remote execution alone, and incremental builds are essentially the same.
+As you can see, the remote-only configuration gave us better results than the local-only configuration for clean builds, but the remote-only configuration became unacceptably slow for incremental builds. As expected, though, dynamic execution won heads-down in all cases: clean builds were many times faster than they are with local or remote execution alone, and incremental builds were essentially the same.
 
-We expect the speedup to be more prominent as builds get larger, as seen in the difference between the iOS and Android builds.
+We expect the speedup to be more prominent as builds get larger and bottlenecked on single actions, as seen in the difference between the iOS and Android builds.
```

Comment:
Which speedup? This sentence is a little ambiguous to me.

I generally expect Remote to get better as builds get larger, since larger builds are generally less critical-path bound and more throughput-bound. But that doesn't sound like what you're saying?

Reply:
The experience with iOS doesn't match your expectation. I think the problem is that larger builds balloon in the number of inputs per action so there end up being more actions that are much more costly (and linking/bundling become even slower). This is a pathological case with iOS support in this particular configuration thoug. Also I think what I had in mind was the impact on incremental builds where we only have the critical path.

But... better remove this altogether. I'm not sure what I'd claim :P